scripts/golangci-lint.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# prints error if a specific command does not exist
+function panic_if_command_not_exists() {
+  if ! command -v "$1" >>/dev/null 2>/dev/null; then
+    echo -e "\033[0;33m$1\033[0m command is not available - please install it first"
+    exit 1
+  fi
+}
```

Comment:
[Notes](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script) on which

Reply:
> [Notes](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script) on which

Yep, although then (again because of `set -e`) we will just get a later error like `bash: command not found: curl`. So it all works and saves a boilerplate function.